.gitlab-ci.yml

```diff
@@ -0,0 +1,36 @@
+include:
+  - project: 'analytics/artificial-intelligence/ai-platform/aip-infrastructure/ci-templates/ci-cd-template'
+    ref: &include_ref 'alexla/AIP-4600'  # TODO AIP-4600 Replace with v2
+    file: '/blocks/python.yml'
+
+stages:
+  - build
+
+build:publish:
+  extends: .aip_python_debian_image
+  stage: build
+  variables:
+    PY_LIBRARY_NAME: 'zillow-kfp'
+    KFP_VERSION_PATH: "sdk/python/kfp/__init__.py"
+  script:
+  - KFP_VERSION=$(cat $KFP_VERSION_PATH | sed -nr "s/^__version__ = ['\"]([^'\"]*)['\"]$/\1/p")
+  - |
+    if [ "${CI_COMMIT_BRANCH}" = "${CI_DEFAULT_BRANCH}" ]; then
+      PY_LIBRARY_VERSION="${KFP_VERSION}.${CI_PIPELINE_IID}"
```

Comment:
Similar [comment](https://github.com/zillow/metaflow/pull/115#discussion_r704840171); would `KFP_VERSION` support semantic versioning?

Reply:
Yep, effectively the same policy as `zillow-metaflow`. In this case I'd expect the `zillow-kfp` version to be `1.7.0.{GITLAB_PIPELINE_ID}`.